packages/protocol/src/peer/PhysicalDeviceProperties.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2022-2025 Matter.js Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Logger } from "#general";
+
+const logger = Logger.get("PhysicalDeviceProperties");
+
+const DEFAULT_SUBSCRIPTION_FLOOR_DEFAULT_S = 1;
+const DEFAULT_SUBSCRIPTION_FLOOR_ICD_S = 0;
+const DEFAULT_SUBSCRIPTION_CEILING_WIFI_S = 60;
+const DEFAULT_SUBSCRIPTION_CEILING_THREAD_S = 60;
+const DEFAULT_SUBSCRIPTION_CEILING_THREAD_SLEEPY_S = 180;
+const DEFAULT_SUBSCRIPTION_CEILING_BATTERY_POWERED_S = 600;
+
+export interface PhysicalDeviceProperties {
```

Comment:
Imaybe PeerDevice instead PhysicalDevice? But I am sure you thought about that a lot :-) so also ok with Pysical

Reply:
Yeah `PeerDeviceProperties` doesn't seem appropriate when this only includes a subset of information we know about a peer, and this is another case of "we only use for peers right now but nothing is really specific to client-side implementation".

Reply:
opk, then we leave as you proposed, fine